contracts/test/integration/vip/vip13.sol

```diff
@@ -0,0 +1,511 @@
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.ERC20ALLOCATOR,
+                arguments: abi.encodeWithSignature(
+                    "disconnectPSM(address)",
+                    MainnetAddresses.VOLT_DAI_PSM
+                ),
+                description: "Disconnect old DAI PSM from the ERC20 Allocator"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.ERC20ALLOCATOR,
+                arguments: abi.encodeWithSignature(
+                    "connectPSM(address,uint248,int8)",
+                    address(voltV2UsdcPriceBoundPSM),
+                    targetBalanceUsdc,
+                    usdcDecimalNormalizer /// 12 decimals of normalization
+                ),
+                description: "Add new USDC PSM to the ERC20 Allocator"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.ERC20ALLOCATOR,
+                arguments: abi.encodeWithSignature(
+                    "connectPSM(address,uint248,int8)",
+                    address(voltV2DaiPriceBoundPSM),
+                    targetBalanceDai,
+                    daiDecimalNormalizer
+                ),
+                description: "Add new DAI PSM to the ERC20 Allocator"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.ERC20ALLOCATOR,
+                arguments: abi.encodeWithSignature(
+                    "connectDeposit(address,address)",
+                    address(voltV2UsdcPriceBoundPSM),
+                    MainnetAddresses.COMPOUND_USDC_PCV_DEPOSIT
+                ),
+                description: "Connect USDC deposit to PSM in ERC20 Allocator"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.ERC20ALLOCATOR,
+                arguments: abi.encodeWithSignature(
+                    "connectDeposit(address,address)",
+                    address(voltV2DaiPriceBoundPSM),
+                    MainnetAddresses.COMPOUND_DAI_PCV_DEPOSIT
+                ),
+                description: "Connect DAI deposit to PSM in ERC20 Allocator"
+            })
+        );
+
+        proposal.push(
+            ITimelockSimulation.action({
+                value: 0,
+                target: MainnetAddresses.PCV_GUARDIAN,
+                arguments: abi.encodeWithSignature(
+                    "addWhitelistAddresses(address[])",
+                    toWhitelist
+                ),
+                description: "Add new DAI, and USDC PSMs to PCV Guardian whitelist"
+            })
+        );
```

Comment:
unset old psms from whitelist -- remove dai and usdc from those psms and send to this new ones